docs/configuration.md

```diff
@@ -1197,6 +1199,7 @@ is unsupported.
 | `remoteurl`| yes     | The URL for the repository on Docker Hub.             |
 | `username` | no      | The username registered with Docker Hub which has access to the repository. |
 | `password` | no      | The password used to authenticate to Docker Hub using the username specified in `username`. |
+| `ttl`      | no      | Expire proxy cache configured in "storage" after this time. Cache 168h(7 days) by default, set to 0 to disable cache expiration, The suffix is one of `ns`, `us`, `ms`, `s`, `m`, or `h`. If you specify a value but omit the suffix, the value is interpreted as a number of nanoseconds. |
```

Comment:
```suggestion
| `ttl`      | no      | Expire proxy cache configured in "storage" after this time. This configuration accepts a numeric value, optionally followed by a [duration unit](https://pkg.go.dev/time#ParseDuration), such as `s` (seconds), `m` (minutes) or `h` (hours). It no unit is specified, the default is `ns` (nanoseconds). Set to `0` to disable cache expiration. The default is 168h (7 days). |
```

Looking at that, it's also very unfortunate that;

- duration does not accept _day_ as unit (if that's likely to be used)
- it also accepts complex formats(?); `1h10m10s`; see https://pkg.go.dev/time#example-ParseDuration
- and floats (`1.234h`)

So somewhat wondering if we should have our own type 🤔

Reply:
I believe we should stay consistent across the codebase. Days can be represented to as multiples of `24h`. If the users find it unwieldy we can come up with some solution to it - there is no risk of breaking anything by sticking with the current approach, IMHO.